.rubocop_todo.yml

```diff
@@ -282,7 +282,7 @@ Metrics/ClassLength:
 
 # Offense count: 1
 Metrics/CyclomaticComplexity:
-  Max: 9
+  Max: 10
```

Comment:
You shouldn't modify this manually, but regenerate the todo file. It is easier :wink: Anyway, I think we should disable this cop for the same reason [I disabled `Metrics/AbcSize` and `PerceivedComplexity`](https://github.com/openSUSE/trollolo/pull/91), we want to keep the project as simple as possible for new contributors. So keep this as it is :wink: